src/Symfony/Component/Serializer/Tests/Encoder/XmlEncoderTest.php

```diff
@@ -12,18 +12,24 @@
 namespace Symfony\Component\Serializer\Tests\Encoder;
 
 use PHPUnit\Framework\TestCase;
-use Symfony\Component\Serializer\Tests\Fixtures\Dummy;
```

Comment:
please revert any change related to ordering: our policy is to add *new* lines in alpha order, but keep old ones as is - to help for merges between branches.